docs/developer-guide/ksqldb-reference/aggregate-functions.md

```diff
@@ -100,6 +100,16 @@ Returns the _approximate_ number of unique values of `col1` in a group.
 The function implementation uses [HyperLogLog](https://en.wikipedia.org/wiki/HyperLogLog)
 to estimate cardinalities of 10^9 with a typical standard error of 2%.
 
+EARLIEST_BY_OFFSET

```

Comment:
Please add a link to the TOC at the top of the page. Thanks!

Reply:
Done